Source/WebKitLegacy/mac/WebView/WebHTMLView.mm

```diff
@@ -609,6 +609,9 @@ - (void)forwardContextMenuAction:(id)sender;
 
     case ContextMenuItemBaseApplicationTag:
         ASSERT_NOT_REACHED();
+    default:
+        return std::nullopt;
+        break;
```

Comment:
Nit - if we go with the `default:` case here, I think we can also remove some of the cases above where we currently return `std::nullopt`.

That said, I think there _might_ be an advantage to the (less succinct) approach of enumerating each of the new types, which is that it would force folks adding new item types in the future to consider (at compile-time) whether or not adding a corresponding `WebMenuItemTag*` is necessary. However, since this is legacy WebKit, the answer should probably be "no" anyways, so either way seems fine!

Reply:
I have a slight preference to explicitly add the new enum values. You could add them underneath `case ContextMenuItemTagLookUpImage:`.

Reply:
Removed the default case and added the new enum values below ContextMenuItemTagLookUpImage